src/states/adult.rs

```diff
@@ -589,6 +589,10 @@ impl Approved for Adult {
         Ok(())
     }
 
+    fn check_vote_status(&mut self) {
+        debug!("{} - Not Required CheckVotingStatus For Adult", self);
```

Comment:
Why? won't it have a different view than the rest of the nodes when reaching elder status.

Reply:
`check_vote_status` only incurs `casting vote of offline`, won't change anything in the `status tracker`.
The `tracker` is affected by add_expectation and add_vote, which is part of the `poll_event` call. Which allows the the adult having the same view as elders'.